### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by FunctionalMetatable**
new middleware
adds a readme
adds /tutorials/featured endpoint (GET)
adds /tutorials/featured endpoint (PUT) (admin-only)

### CODE REVIEW

**Comment by FunctionalMetatable** [comment]
@TheColaber

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+.DS_Store
```
Comment (TheColaber) [inline .gitignore:2]: Everything looks great... the yarn.lock files seems to be changing a lot... Is it supposed to be in the gitignore?

Comment (FunctionalMetatable) [reply .gitignore:2]: if you want it to be it can

Comment (TheColaber) [reply .gitignore:2]: I just want to make sure that doing this won't break anything...

**Pull request merged**